packages/core/src/lib/pages/signin.tsx

```diff
@@ -64,7 +71,17 @@ export default function SigninPage(props: {
           }}
         />
       )}
-      {theme.logo && <img src={theme.logo} alt="Logo" className="logo" />}
```

Comment:
This condition has been removed. Due to this new version, the logo is not showing after th econfiguration